src/app/server/Server.cpp

```diff
@@ -70,6 +73,115 @@ constexpr bool useTestPairing()
 #endif
 }
 
+class ServerStorageDelegate : public PersistentStorageDelegate
+{
+    void SetDelegate(PersistentStorageResultDelegate * delegate) override {}
+    void GetKeyValue(const char * key) override {}
```

Comment:
why are these empty?

What should a 'GetKeyValue' without an output argument and without a return code do?

Reply:
The user of `PersistentStorageDelegate` API can set a `PersistentStorageResultDelegate` to get async response from the storage delegate. We are not using that mechanism here, as the underlying KVS Mgr does not have an async API.

Reply:
I'll add an error log and `chipDie` here, to catch if it gets called in some use case in future.